exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+    "\n",
+    "conn = open_pyexasol_connection_with_lang_definitions(ai_lab_config, compression=True)\n",
+    "conn.execute(\"CREATE SCHEMA SLC_TUTORIAL\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "214aac51-1c9a-4545-935e-aeeed4af55e9",
+   "metadata": {},
+   "source": [
+    "#### Check if your customization did work\n",
+    "We first create a helper UDF which allows us to run arbitrary shell commands inside of a UDF instance. With that we can easily inspect the container."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2036fa12-bd31-4526-a9d6-9b618e1e8c58",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import textwrap\n",
+    "\n",
+    "conn.execute(textwrap.dedent(f\"\"\"\n",
+    "CREATE OR REPLACE {slctmanager.language_alias} SCALAR SCRIPT execute_shell_command_py3(command VARCHAR(2000000), split_output boolean)\n",
+    "EMITS (lines VARCHAR(2000000)) AS\n",
+    "import subprocess\n",
+    "\n",
+    "def run(ctx):\n",
+    "    try:\n",
+    "        p = subprocess.Popen(ctx.command,\n",
+    "                             stdout    = subprocess.PIPE,\n",
+    "                             stderr    = subprocess.STDOUT,\n",
+    "                             close_fds = True,\n",
+    "                             shell     = True)\n",
+    "        out, err = p.communicate()\n",
+    "        if isinstance(out,bytes):\n",
+    "            out=out.decode('utf8')\n",
+    "        if ctx.split_output:\n",
+    "            for line in out.strip().split('\\\\n'):\n",
+    "                ctx.emit(line)\n",
+    "        else:\n",
+    "            ctx.emit(out)\n",
+    "    finally:\n",
+    "        if p is not None:\n",
+    "            try: p.kill()\n",
+    "            except: pass\n",
+    "/\n",
+    "\"\"\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2601c1d-8dd8-4c52-859a-28b7d0d3748b",
+   "metadata": {},
+   "source": [
+    "Check with \"pip list\" if the \"xgboost\" package is installed\n",
+    "We use our helper UDF to run `python3 -m pip list` directly in the container and get the list of currently avaiable python3 packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51d61556-f9f1-4749-b70c-8fc21e141662",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rs=conn.execute(\"\"\"select execute_shell_command_py3('python3 -m pip list', true)\"\"\")\n",
+    "for r in rs: \n",
+    "    print(r[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51cea11e-54ae-4d38-842e-1dac60de0ab5",
+   "metadata": {},
+   "source": [
+    "By running `pip list` directly in the container, we see what is currently available in the container. However, sometimes this might not be what we expected. For these cases, exaslct stores information about the flavor the container was build from within the container.\n",
```

Comment:
```suggestion
    "Running `pip list` inside the container displays the available packages. In case of  unexpected results, please have a look at the information stored by`exaslct` during build-time inside the container.\n",
```

Reply:
Fixed